x-pack/plugins/security_solution/public/common/components/risk_score/risk_score_disabled/host_risk_score_disabled.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { EuiEmptyPrompt, EuiPanel, EuiToolTip } from '@elastic/eui';
+import { useCheckSignalIndex } from '../../../../detections/containers/detection_engine/alerts/use_check_signal_index';
+import { HeaderSection } from '../../header_section';
+import { RiskScoreDocLink } from '../risk_score_onboarding/risk_score_doc_link';
+import { RiskScoreEntity } from '../../../../../common/search_strategy';
+import { RiskScoreEnableButton } from '../risk_score_onboarding/risk_score_enable_button';
+import type { inputsModel } from '../../../store';
+import * as i18n from './translations';
+import { RiskScoreHeaderTitle } from '../risk_score_onboarding/risk_score_header_title';
+
+const EntityAnalyticsHostRiskScoreDisableComponent = ({
```

Comment:
Could we combine this with `EntityAnalyticsUserRiskScoreDisableComponent` and create a generic component that determines the content based on `RiskScoreEntity`, like in `risk_score_deprecated/index.tsx`. I bet we could even combine this component with `risk_score_deprecated` and use one component for all 4 scenarios (`calloutType = 'deprecated' | 'disabled', riskEntity = 'host' | 'user')`. We can do this in follow up PR